lib/runner.ts

```diff
@@ -290,7 +290,7 @@ export class Runner extends EventEmitter {
 
     browser_.forkNewDriverInstance =
         (useSameUrl: boolean, copyMockModules: boolean, copyConfigUpdates = true) => {
-          let newBrowser = this.createBrowser(plugins);
+          let newBrowser = this.createBrowser(plugins, browser_);
```

Comment:
IMO, using the default config or the config of current browser for new browser, both of them make sense.
Creating browser based on parent browser just copy some init data from parent browser, I don't know why setting "waitForAngularEnabled" after creating the browser doesn't work in your case. Perhaps "waitForAngularEnabled" is async, you need wait for it? I am a little hesitate about this since I don't know whether some tests rely on the previous behavior and this change will break their tests.